File: handlers/keyboards.py
from telegram import InlineKeyboardButton, InlineKeyboardMarkup
from config import ADMIN_IDS


def _main_kb(user_id: int) -> InlineKeyboardMarkup:
    rows = [
        [InlineKeyboardButton("📅  View Reminders", callback_data="view:reminders")],
        [InlineKeyboardButton("📝  New Log Entry",  callback_data="log:start")],
        [InlineKeyboardButton("🍳  Browse Recipes", callback_data="recipe:categories")],
    ]
    if user_id in ADMIN_IDS:
        rows.append([InlineKeyboardButton("⚙️  Admin Panel", callback_data="admin:menu")])
    return InlineKeyboardMarkup(rows)


def _back_kb(cb: str = "main:menu", label: str = "« Back") -> InlineKeyboardMarkup:
    return InlineKeyboardMarkup([[InlineKeyboardButton(label, callback_data=cb)]])


def _admin_kb() -> InlineKeyboardMarkup:
    return InlineKeyboardMarkup([
        [InlineKeyboardButton("📅  Reminders",         callback_data="admin:rem:menu")],
        [InlineKeyboardButton("📝  Log Categories",    callback_data="admin:lcat:menu")],
        [InlineKeyboardButton("🍳  Recipe Categories", callback_data="admin:rcat:menu")],
        [InlineKeyboardButton("📖  Add Recipe",        callback_data="admin:rec:add")],
        [InlineKeyboardButton("« Main Menu",           callback_data="main:menu")],
    ])

File: app/Telegram/Keyboards.php
<?php

namespace App\Telegram;

class Keyboards
{
    public static function mainMenu(int $userId): array
    {
        $rows = [
            [
                ['text' => 'Reminders',      'callback_data' => 'view:reminders'],
                ['text' => 'New Log Entry',  'callback_data' => 'log:start'],
            ],
            [
                ['text' => 'Browse Recipes', 'callback_data' => 'recipe:categories'],
            ],
        ];

        if (Guards::isAdmin($userId)) {
            $rows[] = [['text' => 'Admin Panel', 'callback_data' => 'admin:menu']];
        }

        return ['inline_keyboard' => $rows];
    }

    public static function backButton(string $callbackData = 'main:menu', string $label = '« Back'): array
    {
        return ['inline_keyboard' => [[['text' => $label, 'callback_data' => $callbackData]]]];
    }

    public static function adminMenu(): array
    {
        return [
            'inline_keyboard' => [
                [['text' => 'Manage Reminders',        'callback_data' => 'admin:rem:menu']],
                [['text' => 'Manage Log Categories',    'callback_data' => 'admin:lcat:menu']],
                [['text' => 'Manage Recipe Categories', 'callback_data' => 'admin:rcat:menu']],
                [['text' => 'Add Recipe',               'callback_data' => 'admin:rec:add']],
                [['text' => '« Main Menu',              'callback_data' => 'main:menu']],
            ],
        ];
    }
}
